src/technologies/p.json

```diff
@@ -3394,6 +3394,18 @@
     "implies": "Java",
     "website": "https://www.publiccms.com"
   },
+  "PubTech": {
+    "cats": [
+      67
+    ],
+    "description": "PubTech is a consent management platform helping brands and businesses collect, store and leverage their customer consents.",
+    "icon": "PubTech.png",
+    "scriptSrc": [
+      "cmp\\.pubtech\\.ai",
+      
```

Comment:
Do you think it could be useful to extract the version number from the script source?
```suggestion
      "pubtech-cmp-v(.+)\\.js;version:\\1"
```

Reply:
Yes, but I would need to change the regex to handle the advanced SDK case where the version is still 2 but the CMP could be loaded as module in https://cmp.pubtech.ai/2/pubtech-cmp-v2-esm.js.

Reply:
Updated
<img width="1208" alt="image" src="https://github.com/HTTPArchive/wappalyzer/assets/1120896/45ee02a2-0b94-458d-b544-94915d42c778">

Reply:
Thank you so much, this is great.